src/restruct_poc/types.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+namespace oturb {
+
+static constexpr std::size_t LieAlgebraComponents = 6;
+static constexpr std::size_t LieGroupComponents = 7;
+static constexpr std::size_t VectorComponents = 3;
+
+static constexpr double Tolerance = 1.e-16;
+
+using LieAlgebraFieldView = Kokkos::View<double* [LieAlgebraComponents]>;
+using LieGroupFieldView = Kokkos::View<double* [LieGroupComponents]>;
+using VectorFieldView = Kokkos::View<double* [VectorComponents]>;
+
+using View1D = Kokkos::View<double*>;
+using View1D_Vector = Kokkos::View<double[VectorComponents]>;
+using View1D_LieAlgebra = Kokkos::View<double[LieAlgebraComponents]>;
+using View1D_LieGroup = Kokkos::View<double[LieGroupComponents]>;
+
+using View_Rot = Kokkos::View<double[3][3]>;
```

Comment:
I would argue for calling it by the full name - it improves the readability in my opinion and removes any confusion.

`using View_RotationMatrix = Kokkos::View<double[3][3]>;`

Reply:
I don't think I'm using this type anymore. I'll check and remove it. Otherwise, I agree that it should have a more descriptive name.